web/about.html

```diff
@@ -0,0 +1,257 @@
+                <div class="row">
+                    <span style="font-size:40px;cursor:pointer;padding:8px 0px 0px 25px" onclick="openCloseMenu()">&#9776;</span>
+                    <img src="logo.png" alt="Rerum Playground">
+                </div>
+            </div>
+        </header>
+
+        <div class="content">
+            <h2>About RERUM Playground</h2>
+            <p>RERUM Playground is designed to act as a self-guided interactive tutorial for tools involved in working with digital objects.</p>
+            <p>Here, you can explore tools for adding annotations, geolocation data, and many other enhancements to JSON objects without worrying that you may break something.</p>
+            
+            <!--From provided README. Text formatting still needs to be moved to separate stylesheet.-->
+            <h2>What is a JSON object?</h2>
+            <p>JSON (JavaScript Object Notation) is a simple way to store and send information between computers. Think of it like a person's case file, but for data. JSON organizes information using curly braces {}, square brackets [], and colons :. It's easy for both humans and computers to read.</p>
+            <p>For example, a person's details in JSON might look like this:</p>
+            <div class="json">
+                <p>{</p>
+                <p>"name": "Jane Doe",</p>
+                <p>"age": 28,</p>
+                <p>"jobTitle": Software Engineer</p>
+                <p>}</p>
+            </div>
+            <p>JSON is used widely on the internet to send data between a server (like a website's backend) and a user's browser.</p>
+            
+            <h2>What is JSON-LD?</h2>
+            <p>JSON-LD (JSON for Linked Data) is like a more savvier JSON. It does everything JSON does, but adds context to the data. This context helps computers understand the meaning of the data, not just its structure.</p>
+            <p>For instance, when JSON-LD sees "name," it can understand that it's talking about a person's name, not a pet's name or a company's name. This makes it easier for different systems to work together and understand each other's data.</p>
+            <p>JSON-LD has an element called @context, which is like a dictionary that explains what the terms in your data mean. It links your data to standardized definitions on the internet. For example,</p>
+            <div class="json">
+                <p>{</p>
+                <p>"@context": "https://www.com",</p>
+                <p>"name": "Jane Doe",</p>
+                <p>"jobTitle": "Software Developer"</p>
+                <p>}</p>
+            </div>
+            <p>Here, "@context": "https://www.com" tells computers to look up the meanings of "name" and "jobTitle" on www.com. This helps ensure that different systems interpret the data in the same way.</p>
+
+            <!--This description needs to be updated with additional information, possibly including a demo.-->
+            <h2>What is a JSON Manifest?</h2>
```

Comment:
I agree with your comment! I think you need a more thorough explanation of JSON manifests, it's a little brief and doesn't have enough context for understanding.